charts/argo-rollouts/values.yaml

```
@@ -12,8 +11,9 @@ controller:
   tolerations: []
   affinity: {}
   image:
+    registry: quay.io
     repository: argoproj/argo-rollouts
```

Argo has changed the upstream
```suggestion
    repository: quay.io/argoproj/argo-rollouts
```